spec/requests/users_spec.rb

```diff
@@ -1,54 +1,57 @@
 require 'rails_helper'
 
 RSpec.describe "Users", type: :request do
+  fixtures :users
   describe "GET /index" do
     it "returns http success" do
-      get "/users"
+      get users_path
       expect(assigns(:users)).to eq(User.all)
       expect(response).to have_http_status(:success)
     end
   end
 
   describe "GET /show" do
     it "returns http success" do
-      get "/users/show"
+      get user_path(users(:user1).id)
+      expect(assigns(:user)).to eq(users(:user1))
       expect(response).to have_http_status(:success)
     end
   end
 
-  describe "GET /new" do
-    it "returns http success" do
-      get "/users/new"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "POST /create" do
-    it "returns http success" do
-      post "/users/create"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "GET /edit" do
-    it "returns http success" do
-      get "/users/edit"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "PUT /update" do
-    it "returns http success" do
-      put "/users/update"
-      expect(response).to have_http_status(:success)
-    end
-  end
-
-  describe "GET /destroy" do
-    it "returns http success" do
-      get "/users/destroy"
-      expect(response).to have_http_status(:success)
-    end
-  end
+  # describe "GET /new" do
```

Comment:
Entendo que comentou fora porque ainda não implementou de fato esses endpoints, mas acho melhor deixar eles não comentados, afinal eles tão passando e quando for implementar eles, isso vai te ajudar.